.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    "@babel/react",
+    [
+      "@babel/env",
+      {
+        "forceAllTransforms": true
+      }
+    ],
+    "@babel/stage-0"
+  ],
+  "plugins": [
+    "transform-es3-member-expression-literals",
+    "transform-es3-property-literals"
```

Comment:
why do you need these plugins?

Reply:
Webapck 4 related issues.... the good thing about doing it from scratch I know which parts should be removed in the future.